themes/generic/src/main/assets/page.st

```diff
@@ -145,5 +148,7 @@ $endif$
 <script type="text/javascript" src="$page.base$lib/prettify/prettify.js"></script>
 <script type="text/javascript" src="$page.base$lib/prettify/lang-scala.js"></script>
 <script type="text/javascript">jQuery(function(){window.prettyPrint && prettyPrint()});</script>
+<script type="text/javascript">jQuery(function(jq){initOldVersionWarnings(jq, '$page.properties.("project.version")$', '$page.properties.("project.url")$')});</script>
```

Comment:
When the `project.url` is not set in the properties, the `initOldVersionWarnings` function is still called with empty string as a parameter. And then `paradox.json` is looked up at the root of the host. It should not send the request if there is no project url set.